packages/Python/lldbsuite/test/lang/swift/string_index/main.swift

```diff
@@ -0,0 +1,8 @@
+let name = "patatino"
+let index1 = name.firstIndex(of: "t")
+let index_none = name.firstIndex(of: "Q")
+let long_name = "sed avarissimi hominis cupiditati satisfacere posse"
+let index2 = long_name.firstIndex(of: "h")
+print() //%self.expect('frame var -d run-target -- index1', substrs=['(String.Index?) index1 = 2'])
+        //%self.expect('frame var -d run-target -- index_none', substrs=['(String.Index?) index_none = nil'])
+        //%self.expect('frame var -d run-target -- index2', substrs=['(String.Index?) index2 = 15'])
```

Comment:
What about lazily-bridged NSStrings?

Reply:
Do you have a specific example in mind? `firstIndex` doesn't seem to be defined on `NSString`(s)

Reply:
Oh, do you have a test for Substring formatting? Do you want to be printing out "position"?

Reply:
This is just the first part, it's a formatter for StringIndex. I'll build the Substring formatter on top of this.

Reply:
Right, that's what we were talking about. We don't want to display the position, as it's encoding-specific and depends on whether the String was native or lazily bridged.

Reply:
so you're saying that the formatter for `StringIndex` shouldn't exist? That sounds a bit awkward. If people call `firstIndex` and want to see the result, should we show the raw bits? That is a little weird..